week-3/implement/get-card-value.js

```diff
@@ -29,3 +29,24 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+function getCardValue(card) {
+  const cardRank = card.slice(0, -1);
+
```

Comment:
I am a bit confused why you are using slice here. Could you explain please. Maybe I am missing your point of what you are trying to do.

Reply:
i used it to extract the rank of the card from the card string 
For instance, if card is "5♠", card.slice(0, -1) will return "5" because it's taking all characters from index 0 to the last character (-1 index is the last character in the string) excluding the last character ("♠" in this case).